### PR TITLE
7.5.0 release prep

### PR DESCRIPTION
**Pull request by hakbailey**
Release prep for amazon.aws 7.5.0.

### CODE REVIEW

**Review by GomathiselviS — APPROVED** [review]

**Review by gravesm — APPROVED** [review]

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -2692,7 +2692,7 @@ releases:
       - iam_user_info - the ``path`` parameter has been renamed ``path_prefix`` for
         consistency with other IAM modules, ``path`` remains as an alias. No change
         to playbooks is required (https://github.com/ansible-collections/amazon.aws/pull/1933).
-    release_summary: This release includes new features and a bugfix.
+      release_summary: This release includes new features and a bugfix.
```
Comment (abikouo) [inline changelogs/changelog.yaml:2695]: is this expected? Is the YAML file still valid with this update?

Comment (hakbailey) [reply changelogs/changelog.yaml:2695]: It was not valid before, and that release summary section was getting removed from the rst file when running `antsibull-changelog release` or `antsibull-changelog generate`.

Comment (abikouo) [reply changelogs/changelog.yaml:2695]: ok got it

**Review by abikouo — APPROVED** [review]

**Comment by gravesm** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -2777,3 +2777,28 @@ releases:
...
+        ``boto3_resource_list_to_ansible_dict()`` helpers (https://github.com/ansible-collections/amazon.aws/pull/2006).
+      release_summary: This release includes a new feature for the ``iam_user_info``
+        module, bugfixes for the ``cloudwatchlogs_log_group_info`` and ``s3_object``
+        modules and the inventory plugins, and some internal refactoring of ``module_utils``.
+    fragments:
+    - 2006-normalize-refactor.yml
+    - 20240314-cloudwatchlogs_log_group_info-fix-throttling-exceptions.yml
+    - 20240314-s3_object-copy-mode-with-metadata.yml
+    - 20240321-iam-user-info.yml
+    release_date: '2024-04-02'
```
Comment (tremble) [inline changelogs/changelog.yaml:2804]:
```suggestion
    release_date: '2024-04-03'
```

**Review by tremble — APPROVED** [review]

**Comment by tremble** [comment]
recheck

**Pull request merged**